.github/workflows/artifacts.yaml

```diff
@@ -87,7 +90,7 @@ jobs:
     - name: bazel version
       run: bazel version
     - name: Build docker image
-      run: bazel build --sandbox_writable_path=$HOME //docker:orchestration_image_tar
+      run: bazel build --sandbox_writable_path=$HOME //docker:orchestration_image_tar --action_env=PLATFORM=linux/arm64
```

Comment:
Here, future users need to specify the arch when building docker image, though they may not be interested.